src/desktop/apps/artwork/client/index.coffee

```diff
@@ -161,6 +162,7 @@ module.exports =
   init: ->
     setCookie(CLIENT._id)
     recordArtworkView(CLIENT._id, CurrentUser.orNull())
+    splitTest('artwork_sidebar_pageviews').view()
```

Comment:
Cookies for this test have to be set **only** on Artwork pages that have >5 pageviews. We need to ensure 50% split of visitors to that specific subset of artwork pages. So you need to consult the data for that before triggering a view call.

Reply:
oo yea very good catch, nice one!

Reply:
Agreed. Thanks for pointing this out @cavvia !